path_test.go

```diff
@@ -852,3 +1305,266 @@ func Benchmark_Path_matchParams(t *testing.B) {
 		{url: "/api/v1/2005-11-01/peach", params: []string{"2005-11-01", "peach"}, match: true},
 	})
 }
+
+// // go test -race -run Test_RoutePatternMatch
+// func Test_RoutePatternMatch(t *testing.T) {
+// 	t.Parallel()
+// 	type testparams struct {
+// 		url   string
+// 		match bool
+// 	}
+// 	testCase := func(pattern string, cases []testparams) {
+// 		for _, c := range cases {
+// 			match := RoutePatternMatch(c.url, pattern)
+// 			utils.AssertEqual(t, c.match, match, fmt.Sprintf("route: '%s', url: '%s'", pattern, c.url))
+// 		}
+// 	}
+// 	testCase("/api/v1/:param/*", []testparams{
+// 		{url: "/api/v1/entity", match: true},
```

Comment:
Are these tests commented?

Reply:
They were from above and used as a template. They've now been removed.